CodeForBaltimore/Bmore-Responsive
# Updates Dockerfile, docker-compose, and corresponding documentation.

# Description

A few sentences describing the overall goals of the pull request's commits.

## Related PRs

N/A

## Related Issues

List related Issues:

| link |
| -- |
| #289 |

## Todos

None

## Deploy Notes

None

## Steps to Test or Reproduce

1. Run `mkdocs serve` to run local instance of documentation wiki
2. Connect browser to localhost:8000 to view wiki
3. Follow quick start guide simulating a new user/clean machine
4. Confirm quick start is sucessful


## Impacted Areas in Application

- Development
